designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TestingApiHttpService.scala

```diff
@@ -135,31 +157,51 @@ object TestingApiHttpService {
 
   object TestingError {
 
-    final case class TestDataGenerationError(msg: String)             extends TestingError
-    final case class NoScenario(scenarioName: ProcessName)            extends TestingError
-    final case class NoProcessingType(processingType: ProcessingType) extends TestingError
-    final case object NoPermission                                    extends TestingError with CustomAuthorizationError
-    final case class MalformedTypingResult(msg: String)               extends TestingError
+    final case class TestDataGenerationError(msg: String)  extends TestingError
```

Comment:
Wouldn't it be simpler to have a generic error in the endpoint layer with a message? Domain/service errors would be mapped directly to the message, rather than to an endpoint error and then mapping the endpoint error back to the message. WDYT?